### PR TITLE
for editing

### PR DESCRIPTION
**Pull request by ritazakhodyaeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,9 +1,40 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128579957/20.1.10%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T386418)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+#  Dashboard for Web Forms - How to Load and Save Dashboards from/to a Database
 
+This example shows how to create a custom dashboard storage that allows you to store dashboards in a database.
```
Comment (dirk-pieterse) [inline Readme.md]: ` custom dashboard storage` already means `store dashboards`

File: Readme.md [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+* [Global.asax.cs](./CS/Global.asax.cs) (VB: [Global.asax.vb](./VB/Global.asax.vb))
+<!-- default file list end -->
+
+## Example Overview
+
+The example uses the [System.Data.SqlClient](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient?redirectedfrom=MSDN&view=net-5.0) members to connect and operate an MS SQL server database.
+
+A custom dashboard storage should implement one of the following interfaces: [IDashboardStorage](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage) or [IEditableDashboardStorage](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IEditableDashboardStorage).
+
+The following API used in the example:
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
The following API is used in the example:
```

File: Readme.md [hunk 3]
```diff
@@ -1,9 +1,40 @@
...
+<!-- default file list -->
+## Files to Look At
+
+* [DataBaseEditaleDashboardStorage.cs](./CS/DataBaseEditaleDashboardStorage.cs) (VB: [DataBaseEditaleDashboardStorage.vb](./VB/DataBaseEditaleDashboardStorage.vb))
+* [Global.asax.cs](./CS/Global.asax.cs) (VB: [Global.asax.vb](./VB/Global.asax.vb))
+<!-- default file list end -->
+
+## Example Overview
+
+The example uses the [System.Data.SqlClient](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient?redirectedfrom=MSDN&view=net-5.0) members to connect and operate an MS SQL server database.
```
Comment (dirk-pieterse) [inline Readme.md]: Why do you use `operate` here? What does this imply?

File: Readme.md [hunk 4]
```diff
@@ -1,9 +1,40 @@
...
+
+A custom dashboard storage should implement one of the following interfaces: [IDashboardStorage](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage) or [IEditableDashboardStorage](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IEditableDashboardStorage).
+
+The following API used in the example:
+
+- [LoadDashboard](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage.LoadDashboard(System.String)) 
+
+    Returns a dashboard by its ID in the XDocument format, which describes an object model of the dashboard.
```
Comment (dirk-pieterse) [inline Readme.md]: - `Returns a dashboard by its ID` Do you mean 'Returns a dashboard with the corresponding ID'?
- `in the XDocument format` -> in XDocument format
- `which describes` That does this refer to? The dashboard or the format? 'which' is covered in our guidelines

File: Readme.md [hunk 5]
```diff
@@ -1,9 +1,40 @@
...
+- [GetAvailableDashboardsInfo](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage.GetAvailableDashboardsInfo) 
+
+    Returns a list of IDs and Captions of dashboards available in the data storage.
+- [SaveDashboard](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage.SaveDashboard(System.String-System.Xml.Linq.XDocument)) 
+
+    Updates the dashboard with new settings by its id.
```
Comment (dirk-pieterse) [inline Readme.md]: - `with new settings by its id` Again, why do you use 'by something'? This does not make sense
- `id` Should this not be ID?

File: Readme.md [hunk 6]
```diff
@@ -1,9 +1,40 @@
...
+- [AddDashboard](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IEditableDashboardStorage.AddDashboard(System.Xml.Linq.XDocument-System.String)) 
+
+   Takes a dashboard definition with its caption, saves it to the data storage, and returns the ID of a new saved dashboard.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
   Saves a dashboard definition and its caption to the data storage and returns the ID of the new saved dashboard.
```

File: Readme.md [hunk 7]
```diff
@@ -1,9 +1,40 @@
...
+  
+Additionally, this example contains an SQL file ([SavedDashboards.sql](./CS/SavedDashboards.sql)), which can be used to recreate a database used in this example on your side. Do no forget to update the connection string in the **Web.config** file to make it valid in your environment.
```
Comment (dirk-pieterse) [inline Readme.md]: - Again `which`
- `on your side` means what?
- Why do you use `Do no forget`?

**Review by dirk-pieterse — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 5 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
    Saves the specified dashboard with new settings to the dashboard storage.
```

File: Readme.md (hunk 4 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
    Loads a dashboard with the specified ID in XDocument format from the storage. 
```

File: Readme.md (hunk 3 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
The example uses the [System.Data.SqlClient](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient?redirectedfrom=MSDN&view=net-5.0) members to connect and manage an MS SQL server database.
```

File: Readme.md (hunk 7 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
Additionally, this example contains an SQL file ([SavedDashboards.sql](./CS/SavedDashboards.sql)) that can be used to recreate a database used in this example. Update the connection string in the **Web.config** file to make it valid in your environment.
```

File: Readme.md (hunk 1 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
This example shows how to create a custom dashboard storage to load and save dashboards in a database.
```

File: Readme.md (hunk 4 above)
Comment (dirk-pieterse) [reply Readme.md]: Loads a dashboard with the specified ID in XDocument format from storage.

File: Readme.md (hunk 5 above)
Comment (dirk-pieterse) [reply Readme.md:32]: Saves the specified dashboard with new settings to dashboard storage

File: Readme.md (hunk 4 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
    Loads a dashboard with the specified ID in XDocument format from storage.
```

**Pull request merged**